pages/linux/umask.md

```diff
@@ -0,0 +1,15 @@
+# umask
+
+> Display file creation mask or set file creation mask to octal value.
+
+- Display current value as octal:
+
+`umask`
+
+- Display current value symbolically:
+
+`umask -S`
+
+- Turn off permission bits with octal value 'nnnn':
```

Comment:
I'm not advocating for using "mask" in this example description, just in the global description (in order to explain where the command name comes from). What confuses me here is the "turn off" expression -- what do you mean by that? Apart from that, I'm perfectly fine with phrasing the example description with "set permissions".